include/boost/detail/winapi/jobs.hpp

```diff
@@ -10,8 +10,23 @@
 
 #include <boost/detail/winapi/basic_types.hpp>
 #include <boost/detail/winapi/access_rights.hpp>
+#include <boost/detail/winapi/detail/cast_ptr.hpp>
 
-#if !defined( BOOST_USE_WINDOWS_H )
+// Microsoft documentation states all of these APIs are available
+// in Windows XP / Windows 2003 or later
+
+#if (BOOST_USE_WINAPI_VERSION >= BOOST_WINAPI_VERSION_WINXP)
+
+#if (BOOST_USE_WINDOWS_H)
+// IsProcessInJob is defined in jobapi.h in the Windows 8 SDK or later
+#if (BOOST_WINAPI_WINDOWS_SDK >= BOOST_WINAPI_WINDOWS_SDK_8)
+#include <jobapi.h>
```

Comment:
Why are these includes needed? Both jobapi.h and jobapi2.h are included from WinBase.h, which is included from windows.h.

Reply:
The Microsoft documentation for IsProcessInJob (https://msdn.microsoft.com/en-us/library/windows/desktop/ms684127(v=vs.85).aspx) indicates this is in a different header for certain SDK revisions, but I see it in winbase up through the latest SDK 10 so I will remove these sections.